content/en/blogs/TiDB-on-KubeSphere-using-qke.md

```diff
@@ -8,13 +8,15 @@ author: 'Willqy, Feynman, Sherlock'
 snapshot: 'https://ap3.qingstor.com/kubesphere-website/docs/tidb-architecture.png'
 ---
 
+![](https://ap3.qingstor.com/kubesphere-website/docs/20201028212049.png)
+
 In a world where Kubernetes has become the de facto standard to build application services that span multiple containers, running a cloud-native distributed database represents an important part of the experience of using Kubernetes. In this connection, [TiDB](https://github.com/pingcap/tidb), as an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads, has come to my awareness. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability. It strives to provide users with a one-stop database solution that covers OLTP (Online Transactional Processing), OLAP (Online Analytical Processing), and HTAP services. TiDB is suitable for various use cases that require high availability and strong consistency with large-scale data.
```

Comment:
Please delete "as" in "as an open-source NewSQL database that supports..."

I wrote this sentence though, "as" is a bit redundant here when I read it twice.